stdlib/public/core/StringBridge.swift

```diff
@@ -516,17 +516,8 @@ private func formConstantTaggedCocoaString(
 #if !arch(arm64)
   return nil
 #else
-
-  let constantPtr:UnsafeRawPointer = Builtin.reinterpretCast(untaggedCocoa)
-
-  // Check if what we're pointing to is actually a valid tagged constant
-  guard _swift_stdlib_dyld_is_objc_constant_string(constantPtr) == 1 else {
-    return nil
-  }
-
-  let retaggedPointer = UInt(bitPattern: constantPtr) | expectedConstantTagValue
-
-  return unsafeBitCast(retaggedPointer, to: AnyObject.self)
+// See git history if we ever need the implementation of this again
+  return nil
```

Comment:
Should we remove `expectedConstantTagValue` as well? Is `constantTagMask` dead as well?

Reply:
Should we eliminate the `#if` conditional entirely?

Reply:
I updated the diff to remove everything related to this rather than just the slow bit